src/metabase/models/channel.clj

```diff
@@ -40,7 +40,14 @@
   (assert-channel-type instance)
   instance)
 
+(defn- delete-pulse-channels [channel-id channel-type]
+  (t2/delete! :model/PulseChannel
+              :channel_id channel-id
+              :channel_type (-> channel-type keyword name)))
```

Comment:
Why can't we just delete by channel id? i.e.
```
(defn- delete-pulse-channels [channel-id channel-type]
  (t2/delete! :model/PulseChannel :channel_id channel-id))
```

Reply:
yea on second thought it's better that way dc8e3399d2